src/DNSC/Concurrent.hs

```diff
@@ -0,0 +1,45 @@
+module DNSC.Concurrent (
+  forkProcessQ,
+  forkLoop,
+  forksProcessQ,
+  forksLoop,
+  )where
```

Comment:
should put a space char after `)`.

Reply:
https://github.com/khibino/dns-resolver/pull/4/commits/3ccbb0d7c80c74abf80d237c75dca3aed134953a